Assets/AltTester/Editor/Logging/EditorLogManager.cs

```diff
@@ -0,0 +1,48 @@
+/*
+    Copyright(C) 2023 Altom Consulting
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program. If not, see <https://www.gnu.org/licenses/>.
```

Comment:
@andresilveirah @bohdan-go-wombat are we okay with adding GNU-licensed software to our repo?